mountinfo/mountinfo_bsd.go

```diff
@@ -54,3 +54,11 @@ func mounted(path string) (bool, error) {
 	// Fallback to parsing mountinfo
 	return mountedByMountinfo(path)
 }
+
+func getMountinfo(entry *unix.Statfs_t) *Info {
+	var mountinfo Info
+	mountinfo.Mountpoint = unix.ByteSliceToString(entry.Mntonname[:])
+	mountinfo.FSType = unix.ByteSliceToString(entry.Fstypename[:])
+	mountinfo.Source = unix.ByteSliceToString(entry.Mntfromname[:])
```

Comment:
were these renamed at some point ? looks like the current names are;

```suggestion
	mountinfo.Mountpoint = unix.ByteSliceToString(entry.F_mntonname[:])
	mountinfo.FSType = unix.ByteSliceToString(entry.F_fstypename[:])
	mountinfo.Source = unix.ByteSliceToString(entry.F_mntfromname[:])
```

https://github.com/golang/sys/blob/bc2c85ada10aa9b6aa9607e9ac9ad0761b95cf1d/unix/ztypes_openbsd_amd64.go#L99-L102

Reply:
Oh! I see; that's for the other bsd types, but for darwin (macOS) and freebsd it's the previous names 

- https://github.com/golang/sys/blob/bc2c85ada10aa9b6aa9607e9ac9ad0761b95cf1d/unix/ztypes_darwin_amd64.go#L100-L102
- https://github.com/golang/sys/blob/bc2c85ada10aa9b6aa9607e9ac9ad0761b95cf1d/unix/ztypes_freebsd_amd64.go#L129-L131


so you'll need multiple implementations for this function

Reply:
Oh, thanks, good catch.